rfc822/header.go

```diff
@@ -270,15 +270,23 @@ func mergeMultiline(line []byte) string {
 	var res [][]byte
 
 	for line := range forLines(bufio.NewReader(bytes.NewReader(line))) {
-		res = append(res, bytes.TrimSpace(line))
+		trimmed := bytes.TrimSpace(bytes.Trim(line, "\t"))
```

Comment:
isn't TrimSpace also trimming `\t` ?

Reply:
Looks like you are correct. The change must have masked some  other issue I was having at the time. Good catch.